astropy/coordinates/builtin_frames/utils.py

```diff
@@ -85,7 +76,7 @@ def get_polar_motion(time):
 
     if np.any(status == iers.TIME_BEYOND_IERS_RANGE):
         wmsg = ('Tried to get polar motions for times after IERS data is '
-                'valid. Defaulting to polar motion from the 50-yr mean for those.' + _IERS_HINT)
+                'valid. Defaulting to polar motion from the 50-yr mean for those.')
```

Comment:
Maybe add to this warning something along the lines of "this may affect precision at the 10s of arcsec level"?  (Because that information was previously in the hint.)